monitoring/README.md

```diff
@@ -37,68 +37,34 @@ The first step is to update all the pis' package repositories, install all depen
 
 ### On the VM
 
-1. Make the /usr/bin/piman directory and copy the monitoring directory there:
+1. Clone the piman directory into /usr/local:
 
 ```
-sudo mkdir /usr/bin/piman
-sudo cp -r /path/to/piman/monitoring /usr/bin/piman
+cd /usr/local
+sudo git clone https://github.com/SJSU-CS-systems-group/piman.git
 ```
+This will make sure that /usr/local/piman/monitoring is a path which exists on your system.
 
-2. Copy the hosts.csv file to /usr/bin/piman:
-
-```
-sudo cp /path/to/piman/hosts.csv /usr/bin/piman
-```
-
-3. Copy the monitoring.service and grafana.service files to /etc/systemd/system:
+2. Copy the monitoring.service and grafana.service files to /etc/systemd/system:
 
 ```
 sudo cp monitoring.service grafana.service /etc/systemd/system
 ```
 
-4. Install python dependencies:
+3. Install python dependencies:
 
 ```
 sudo pip3 install requests
 ```
 
 ### On the Pis
-1. Copy the files monitoring-server.py and pi-server-monitoring.service to each pi's home directory:
-
-```
-cd /path/to/piman/monitoring
-scp pi-server-monitoring.service monitoring-server.py pi@<ip address>:
-```
-where `<ip address>` is the ip address of the pi.
-
-Repeat this for all other pis.
-
-2. SSH into one of the pis:
-
+1. Run the following script to initialize the pis, copy all needed files to them, and install and run the monitoring service on them:
 ```
-ssh pi@<ip address>
+cd /usr/local/piman/monitoring
+sudo ./init_pis.sh pi piman /usr/local/piman/monitoring/hosts.csv /usr/local/piman/monitoring
```

Comment:
this doesn't seem correct. isn't hosts.csv in /usr/local/piman?

Reply:
It is indeed. I fixed the path to be /usr/local/piman/hosts.csv.